topic04-git-introduction/index.html

```diff
@@ -173,6 +177,30 @@ <h2>  Git Introduction </h2>
 </div>
 
           
+          
```

Comment:
What's the difference between index.html and indexmodule.html

Reply:
@laurafitzgerald It seems like the index.html contains all the contents of the page and the indexmoodle.html seems to contain the contents of the card.

Reply:
@JameelB thanks for that.